src/server/editorServices.ts

```diff
@@ -444,10 +444,10 @@ namespace ts.server {
             if (!this.eventHandler) {
                 return;
             }
-            this.eventHandler(<ProjectLanguageServiceStateEvent>{

```

Comment:
The casts in this file aren't required at all if you change `ProjectServiceEventHandler` to use overloads instead of a single overload with a union for its parameter type. That's probably a nicer solution.